its/tests/src/test/java/its/SonarQubeCommunityEditionTests.java

```diff
@@ -152,6 +154,19 @@ static void createSonarLintUser() {
     adminWsClient.users().create(new CreateRequest().setLogin(SONARLINT_USER).setPassword(SONARLINT_PWD).setName("SonarLint"));
   }
 
+  @Test
+  void test_revoke_token() {
+    var tokenName = "testTokenForRevoking";
+    var testToken = adminWsClient.userTokens()
+      .generate(new GenerateRequest().setName(tokenName));
+    assertThat(testToken.getName()).isEqualTo(tokenName);
+
+    // Using the credentials we used throughout the test, we want to remove the additional token!
+    backend
+      .getUserTokenService()
+      .revokeToken(new RevokeTokenParams(ORCHESTRATOR.getServer().getUrl(), testToken.getName()));
```

Comment:
is an assertion missing here? 🤔

Reply:
Yes you're right, I did not commit the changes here.